src/System.CodeDom/tests/System/CodeDom/Compiler/CompilerResultsTests.cs

```diff
@@ -30,20 +30,59 @@ public void Ctor_TempFileCollection(TempFileCollection tempFiles)
         }
 
         [Fact]
-        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "Assembly.Load ignores CodeBase by design, except on netfx. See https://github.com/dotnet/coreclr/issues/10561")]
-        public void CompiledAssembly_ValidPathToAssembly_ReturnsExpected()
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, ".NET Core ignores AssemblyName.CodeBase")]
```

Comment:
Isn't this test now will work on any framework?

Reply:
I'm confused by this comment: Comment suggest it should be disabled on .NET Core but attribute suggest it will be disabled on full framework